File: src/main/programs/Java/LeapYear.java
package Java;

import java.util.Scanner;

public class LeapYear {
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        int yr= sc.nextInt();
        if (yr%4==0 && yr%100!=0){
            System.out.println("Leap Year");
        }/*else if (yr%100==0 && yr%400==0){
            System.out.println("Leap Year");
        }else if (yr%400==0){
            System.out.println("Leap Year");
        }*/else {
            System.out.println("Not Leap year");
        }
    }
}

File: src/main/programs/logicbuilding/LeapYear.kt
package logicbuilding

fun main() {
    var year=2024
    if(year%4==0)
        println("Leap Year")
    else if (year%100==0)
        println("Not a Leap Year")
    else if (year%400==0)
        println("Leap Year")
    else
        println("Not a Leap Year")
}